<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryDialog</class>
 <widget class="QDialog" name="QueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Query Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="EditBox"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTableView" name="View"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="ResponseLabel">
     <property name="text">
      <string>Sql Server response:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="LastQueryButton">
       <property name="toolTip">
        <string>Previous Query</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/go-last-rtl.png</normaloff>:/new/prefix1/go-last-rtl.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SendButton">
       <property name="text">
        <string>Send query</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>EditBox</tabstop>
  <tabstop>View</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QueryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QueryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SendButton</sender>
   <signal>clicked()</signal>
   <receiver>QueryDialog</receiver>
   <slot>onSendQuery()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>858</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LastQueryButton</sender>
   <signal>clicked()</signal>
   <receiver>QueryDialog</receiver>
   <slot>onBackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSendQuery()</slot>
  <slot>onBackButton()</slot>
 </slots>
</ui>
